contracts/interfaces/ILimitOrder.sol

```diff
@@ -32,12 +32,7 @@ interface ILimitOrder is IStrategyBase {
     /// @param recipient The address of the recipient which will receive tokens from maker
     /// @param fillReceipt Contains details of this single fill
     event LimitOrderFilledByTrader(
-        bytes32 indexed orderHash,
-        address indexed maker,
-        address indexed taker,
-        bytes32 allowFillHash,
-        address recipient,
-        FillReceipt fillReceipt
+        bytes32 indexed orderHash, address indexed maker, address indexed taker, bytes32 allowFillHash, address recipient, FillReceipt fillReceipt
```

Comment:
This creates different result from `prettier` and `fmt`: `prettier` has a problem but `fmt` does not, though this line does not exceed 160 characters...so I'm not sure why `prettier` is complaining.

Reply:
And this difference from `fmt` results in it updating many function and event declarations. This one is one of the many examples.